cmd/controller/app/options/options.go

```diff
@@ -34,7 +34,7 @@ import (
 
 type DevOpsControllerManagerOptions struct {
 	KubernetesOptions *k8s.KubernetesOptions
-	JenkinsOptions    *jenkins.Options
+	DevOpsOptions     *jenkins.Options
```

Comment:
It should be JenkinsOption instead of DevOpsOption. Because it only has Jenkins options.